plot_continuous_monitor.py

```diff
@@ -137,7 +139,9 @@ def plot_continuous_monitor(filename, open_graphs=False):
       xvdb_read_throughput,
       xvdb_write_throughput,
       xvdf_running_disk_monotasks,
-      xvdb_running_disk_monotasks]
+      xvdb_running_disk_monotasks,
+      free_heap_memory / (1.074 * pow(10, 9)),
```

Comment:
Can you define a BYTES_PER_GIGABYTE as 1024*1024*1024 somewhere, and then use that here?